index.html

```diff
@@ -523,6 +523,44 @@ <h3 class="modal-title" id="myModalLabel" style="margin-left: 150px;"></h3>
 					}
 				]
 			},
+			{
+				"data-thumbnail": "images/Mubaris.jpg",
+				"data-name": "Mubaris NK",
+				"data-desig": "Member",
+				"data-github": "https://github.com/mubaris",
+				"data-link-facebook": "https://www.facebook.com/mubaris.hassan.7",
+				"data-twitter": "https://twitter.com/MubarisHassan",
```

Comment:
Don't remove linked in link. Better make the link "#", if you don't have any link.